fqd_prog.c

```diff
@@ -70,3 +70,16 @@ fqd_route_prog__payload_prefix__s(fq_msg *m, int nargs, valnode_t *args) {
     return true;
   return false;
 }
+
+bool
+fqd_route_prog__payload_contains__s(fq_msg *m, int nargs, valnode_t *args) {
+    int flen, i;
+    fq_assert(nargs == 1);
+    fq_assert(args[0].value_type == RP_VALUE_STRING);
+    flen = strlen(args[0].value.s);
+    if(flen > m->payload_len) return false;
+    for(i=0;i<=m->payload_len - flen;i++)
```

Comment:
That code was copied from the `route_contains` program. I'll change it in both places.